runtime/onert/backend/train/TensorBuilder.h

```diff
@@ -63,6 +63,7 @@ class TensorBuilder
   void notifyFirstUse(const ir::OperandIndex &);
   void notifyLastUse(const ir::OperandIndex &);
   void notifyBackwardFirstUse(const ir::OperandIndex &);
+  void notifyBackwardLastUse(const ir::OperandIndex &);
```

Comment:
This member function will be used in `TensorPlanner` to plan memory of backwarding tensors(back-propagated and gradient).